config/initializers/devise.rb

```diff
@@ -1,7 +1,15 @@
 # License: AGPL-3.0-or-later WITH Web-Template-Output-Additional-Permission-3.0-or-later
 # Use this hook to configure devise mailer, warden hooks and so forth.
 # Many of these configuration options can be set straight in your model.
+
+# Set expiration for OTP in seconds
+Devise.otp_allowed_drift = 900 # 15 minutes
```

Comment:
How about this instead?

```suggestion
Devise.otp_allowed_drift = 15.minutes
```